common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerRequest.java

```diff
@@ -219,4 +221,8 @@ private static final class SerializedNames {
     @SerializedName(SerializedNames.POWER_OPT_CHECK_ENABLED)
     private boolean mPowerOptCheckEnabled;
 
+    @Nullable
+    @SerializedName(SerializedNames.SPAN_CONTEXT)
+    private SerializableSpanContext mSpanContext;
```

Comment:
More so a question for my own knowledge: I understand the use case for serialization in something like PublicClientApplicationConfiguration in Msal, as it allows us to read a configuration json to create a PublicClientApplicationConfiguration object. What is the use case as far as the open telemetry/span implementation? Is serialization in this case used to write to a json to capture the span's telemetry?

Reply:
The use case here is about sending data over IPC from one app/process to another app/process. A complex object needs to be represented in a type such as String or byte to be able to be sent across processes. This is achieved through serialization and we are using that here to convert it to a String so we can send it from MSAL to Broker over IPC.